### ISSUE TITLE
seems not converted in ./generated floder?

### ISSUE THREAD

**Issue by HudsonHuang** [r9y9/gantts#7]
Hi Ryuichi,
I am trying to reproduce your project "**gantts**": https://github.com/HudsonHuang/gantts
To convert clb to slt, with your code I run like this:

#source activate tensorflow
#export LD_LIBRARY_PATH=/usr/local/cuda/lib64/
#pip install -U pip
#conda install pytorch torchvision cuda80 -c soumith
#pip install http://download.pytorch.org/whl/cu80/torch-0.2.0.post3-cp35-cp35m-manylinux1_x86_64.whl 
#pip install torchvision
python setup.py install
bash ./vc_demo.sh awb-clb1 /home/lab-huang.zhongyi/data/cmu_arctic clb awb

After training, I got files in ./generated, but it seems not converted, **it's still clb's voice**, did I mistaken in some steps? 
[clb-awb1(1).zip](https://github.com/r9y9/gantts/files/1485243/clb-awb1.1.zip)
[vc_gan_test2(1).zip](https://github.com/r9y9/gantts/files/1485244/vc_gan_test2.1.zip)

PS. I checked tensorboard and the Loss is dropping normalliy:
![_20171119103626](https://user-images.githubusercontent.com/5734717/32986892-a64ecc12-cd16-11e7-861a-037d496ac028.png)

and, I modified **vc_demo.sh**, added to parameters to support person modifiy:https://github.com/HudsonHuang/gantts/blob/master/vc_demo.sh
I hope it doesn't matter.

This is an excellent work , thank you!

**Comment by r9y9** [comment]
1. Voice conversion based on spectrum differenceial ([ref](https://www.researchgate.net/publication/287247914_Statistical_singing_voice_conversion_with_direct_waveform_modification_based_on_the_spectrum_differential)) cannot be simply applied to cross-gender voice conversion. You may want to remove ``--diffvc`` ([here](https://github.com/r9y9/gantts/blob/8111e255fb65ded1f0361c18c1a7e5c4c13ad840/vc_demo.sh#L56)) if you want female to male voice conversion.
2. I didn't implement F0 conversion for simplicity. You probably need to implement it (e.g., linear transformation).

**Comment by HudsonHuang** [comment]
Thanks  for your reply!
But I try many times to run your orginal code with:

/vc_demo.sh orginal_test ../../data/cmu_arctic/

Which was **from clb to slt **by default, but still doen't converted. I push my result and the checkpoints to [here](https://github.com/HudsonHuang/question_for_gantts), could your please check it out?

**Comment by r9y9** [comment]
For clb to slt, audio samples can be found at http://nbviewer.jupyter.org/github/r9y9/gantts/blob/master/notebooks/Test%20VC.ipynb. Did you compare them and find some differences?

**Comment by HudsonHuang** [comment]
Oh, thank you, I it seems the result of your notebook and my run was the same.

**Comment by r9y9** [comment]
I think I can close this.
